src/objective-c/change-comments.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+
+import re
+import sys
+
+
+if len(sys.argv) < 2:
+  print("Please provide at least one source file name as argument.")
+  quit()
```

Comment:
Wow. Okay, I'd never before ever heard of `quit`, but it looks like it's documented at https://docs.python.org/2/library/constants.html#quit. Wow.